### ISSUE TITLE
Error with heads when using the skinsrestorer plugin

### ISSUE THREAD

**Issue by Flaimer2** [InsinuateProjects/TrMenu#37]
**Describe the bug**
When you set head: name or placeholder with the skinsrestorer plugin, it shows Steve's head, even if you write head: 'hypixel' it will show Steve's head.

**To Reproduce**
1. Install TrMenu, SkinsRestorer
2. Set the value of material: 'head: hypixel'
3. We see a bug with the head

**Expected behavior**
I was expecting to see what it says in the wiki: Full support for SkinsRestorer has been implemented, allowing access to player-displayed skins

**Server Information**
```YAML
TrMenu Dump Information (Date: 2021-08-17 21:08:05)

| Server OS: Windows 10 amd64 10.0
| Server software: git-Paper-1618 (MC: 1.12.2) (1.12.2-R0.1-SNAPSHOT)
| Java version: 11.0.11

| TabooLib: 6.x
| TrMenu: 3.0-PRE-18
Installed Plugins: 
· BetterSocial - 1.3-BETA (0 KB)
· BlockParticles - 1.12-Build#15 (0 KB)
· ConditionalEvents - 3.7.1 (0 KB)
· DisableJoinMessage - 1.0 (0 KB)
· FutureHolograms - 1.0.3 (0 KB)
· HolographicDisplays - 2.4.6 (0 KB)
· InteractiveBoard - 13.5 (0 KB)
· InteractiveBooks - 1.5.3 (0 KB)
· ItemJoin - 5.2.3-SNAPSHOT-b754 (0 KB)
· LobbyPresents - 4.3 (0 KB)
· LuckPerms - 5.2.40 (0 KB)
· MainChairs - 0.9.4 (0 KB)
· NBTAPI - 2.8.0 (0 KB)
· NoteBlockAPI - 1.6.1-SNAPSHOT (0 KB)
· PlaceholderAPI - 2.10.10 (0 KB)
· ProtocolLib - 4.5.0 (0 KB)
· RainbowBlocks - 13.0 (0 KB)
· SkinsRestorer - 14.1.3 (0 KB)
· SuperLobby - 4.3.5 (0 KB)
· TAB - 2.8.11-pre17 (0 KB)
· TrHologram - 2.2 (0 KB)
· TrMenu - 3.0-PRE-18 (0 KB)
· Vault - 1.7.1-b${env.TRAVIS_BUILD_NUMBER} (0 KB)
· XConomy - 2.27 (0 KB)
· eGlow - 3.1.0 (0 KB)
```

**Comment by Score2** [comment]
Try to use latest build.:)

**Comment by Flaimer2** [comment]
Heads don't work in the plugin
![2021-08-20_11 48 36](https://user-images.githubusercontent.com/63396386/130207293-5f8f2130-d39d-4ba3-9758-52066f729445.png)
https://paste.helpch.at/ehihonijaw.sql

**Comment by Flaimer2** [comment]
@Score2 up

**Comment by Flaimer2** [comment]
Heads does not work (I have the latest version of SkinsRestorer and TrMenu)

**Comment by 157946a** [comment]
Are you bc server, is bungeecord mode also turned on, or is it single-ended?

**Comment by Flaimer2** [comment]
> Are you bc server, is bungeecord mode also turned on, or is it single-ended?

Yes I use bungeecord and in SkinsRestorer everything is configured under BungeeMode

**Comment by 157946a** [comment]
Turn on the bc mode, and then delete the skins of all subservers, leaving only the skins of bc

**Comment by Flaimer2** [comment]
> Turn on the bc mode, and then delete the skins of all subservers, leaving only the skins of bc

I have the same thing.

**Comment by 157946a** [comment]
Do you only leave the skin plug-in of bc? I have solved similar problems for others before
https://github.com/InsinuateProjects/TrMenu/issues/76

**Comment by Flaimer2** [comment]
> Do you only leave the skin plug-in of bc? I have solved similar problems for others before #76

I removed the plugin from BungeeCord and only left it on the spigot servers, it worked for me, so we can close
